### PR TITLE
M3 4683 update cypress

### PR DESCRIPTION
**Pull request by ndukatz**
The main changes in the test files deal with how the stubbing/modifying of requests/responses works in this cypress update. I also cleaned up and refactored some other things. The yarn lock is what I am the least sure about as far as if I did it correctly, but cypress 6.2.1 is working on this branch.

**How to verify that that the tests pass:**
- yarn up
- yarn cy:e2e **(runs headless)**
- Result should be "✔  All specs passed!"

Make sure your .env contains:
REACT_APP_LOGIN_ROOT='https://login.linode.com'
REACT_APP_API_ROOT='https://api.linode.com/v4'
REACT_APP_APP_ROOT='http://localhost:3000'
REACT_APP_LAUNCH_DARKLY_ID=(CMR flag)
REACT_APP_CLIENT_ID=(https://cloud.linode.com/profile/clients, copy ID)
MANAGER_OAUTH=(create in https://cloud.linode.com/profile/tokens > "Add a Person Access Token", Save the token that appears in the modal, you will only see it once)

### CODE REVIEW

**Comment by ndukatz** [comment]
Looks like the tests are failing because of the yarn lock file being changed... not sure if that means I did it incorrectly or not.

**Comment by Jskobos** [comment]
Seems like it may be similar to this: https://github.com/cypress-io/cypress/issues/6789#issuecomment-605629069 Try bumping to the most recent version of testing-library/cypress

**Comment by ndukatz** [comment]
@Jskobos that may have done the trick?

**Review by Jskobos — APPROVED** [review]
On my full run change-username.spec failed, but it passed in isolation so 🤞 it was probably just a fluke.

File: packages/manager/cypress/integration/helpAndSupport/open-support-ticket.spec.ts [hunk 1]
```diff
@@ -11,90 +17,72 @@ describe('help & support', () => {
...
+        entity: null,
+        id: ticketId,
+        opened: ts.toISOString(),
+        opened_by: user,
+        status: 'new',
+        summary: 'cy-test ticket',
+        updated: ts.toISOString(),
+        updated_by: user
+      };
+      // intercept create ticket request, stub response.
```
Comment (johnwcallahan) [inline packages/manager/cypress/integration/helpAndSupport/open-support-ticket.spec.ts:40]: Something to consider in the future (not in this PR): 

I wonder if it's worth creating reusable functions for these intercepts in order to make these clearer. Maybe something like:

```ts
// Without stubbing:
interceptAndSpy({ 
  method: 'POST',
  route: '*/support/tickets', 
  as: 'createTicket'
});

// With stubbing:
interceptAndStub({ 
  method: 'POST', 
  route: '*/support/tickets', 
  stub: mockTicketData, 
  as: 'createTicket'
});
```

It'd make the comments unnecessary, etc. Just a drive by thought to consider later.

**Review by johnwcallahan — COMMENTED** [review]
Everything is passing except the first (change-username), which passes in isolation (like Jared's experience).

When running in isolation the Username input is initially empty. When running in sequence, the input is pre-filled with the username, so the test fails.

File: packages/manager/cypress/integration/helpAndSupport/open-support-ticket.spec.ts (hunk 1 above)
Comment (ndukatz) [reply packages/manager/cypress/integration/helpAndSupport/open-support-ticket.spec.ts:40]: I like that idea

**Review by johnwcallahan — APPROVED** [review]
Username test is passing now 👍

**Pull request merged**